crypto/http/http_local.h

```diff
@@ -15,17 +15,6 @@
 
 /* name aliases for legacy names with name prefix "OCSP_" */
 typedef OCSP_REQ_CTX OSSL_HTTP_REQ_CTX;
```

Comment:
Is there a reason you're leaving this typedef alone? Following your argument, it should be just as wrong as the function aliases you removed.

Reply:
Yeah, I thought about that and probably made the wrong discussion.  Additional commit coming up.